addon/components/frost-checkbox.js

```diff
@@ -76,18 +67,59 @@ export default Component.extend(PropTypeMixin, {
     }
   },
 
-  // == Events ================================================================
+  // == DOM Events ============================================================
 
+  /**
+   * Handle the focusIn event
+   * @params {Event} e - the event being handled
+   */
   _onFocus: Ember.on('focusIn', function (e) {
     // If an onFocus handler is defined, call it
     if (this.attrs.onFocus) {
       this.attrs.onFocus()
     }
   }),
 
+  /**
+   * Handle the keypress event
+   * @param {Event} e - the event being handled
+   * @returns {Boolean|undefined} false if we want to stop propagation
+   */
+  keyPress (e) {
+    if (e.keyCode === 32) {
+      if (get(this, 'disabled') !== true) {
+        this.$('input').prop('checked', !this.$('input').prop('checked'))
+        this.send('input')
+      }
+      e.preventDefault()
+      e.stopPropagation()
+      return false
+    }
+  },
+
+  // == Lifecycle Hooks =======================================================
+
+  /* Ember.Component method */
```

Comment:
First, it's not JSDoc `/*` vs `/**` if it *were* JSDdoc, it'd fail lint, b/c we don't discuss params (well, I'm not sure on this particular one b/c I'm too lazy to look at the file view to see which function this is talking about, but you get the idea). The original reason for those comments was b/c our previous team wanted all functions to have *some* header, b/c it's easy to see in review that a function is missing a header and complain than to have to figure out if this particular function should have a header or not. It was particularly useful when there weren't sections w/i the file delimited by comment blocks, or when more than one type of function was allowed within the same block. I agree that now that we have a section dedicated to these methods, it's less useful. Something about seeing an undocumented function still triggers my PR review senses though... probably b/c when doing a review of a new function being added, you don't get the context of what block it's in. I suppose I can get over that if everyone feels it's harmful to have them.